### PR TITLE
hash game

### PR DESCRIPTION
**Pull request by kajarenc**
review please

### CODE REVIEW

File: Assignment_4/Karen/hash_game.py [hunk 1]
```diff
@@ -0,0 +1,38 @@
+class mySet():
+    def __init__(self):
+        self.my_set = set()
+        self.p = 31
+        self.q = 2049982463
```
Comment (tigrandp) [inline Assignment_4/Karen/hash_game.py]: You should make calculations modulo `2^64`, otherwise it is not a fair game.

**Comment by tigrandp** [comment]
@kajarenc As I said you should have imported `/Assignment_4/Python/hash_table.py` and just provide hash function.

File: Assignment_4/Karen/hash_game.py (hunk 1 above)
Comment (kajarenc) [inline Assignment_4/Karen/hash_game.py]: but on the last lecture you said that the size of the table should be prime ? also if it fixed 2^64 , why HashTable class have mod parameter ?

**Comment by kajarenc** [comment]
make some changes again

File: Assignment_4/Karen/hash_game.py (hunk 1 above)
Comment (tigrandp) [inline Assignment_4/Karen/hash_game.py]: Yes I said if you want 'good' hash table you should use prime number. But the assignment was to create simple open addressed hash_table which is using some `modulo` where `modulo` is a power of 2, in this case it is much easier to break that hash_table. That was a game right ?

And I've already created the interface which you should use, just import that in your program and provide hash_function.

**Comment by tigrandp** [comment]
@kajarenc did you stuck or what is reason of delay ?

**Comment by kajarenc** [comment]
i don't understand how i can see your LGTM, and when i  can merge the changes

**Comment by tigrandp** [comment]
@kajarenc : but you didn't change the way I said. you can take a look on Gnun's solution which he already pushed.

**Comment by kajarenc** [comment]
ok, the only difference is  mod = 1048576 , yes ? what wrong with my function

**Comment by tigrandp** [comment]
@kajarenc sorry my bad. feel free to merge.

**Comment by kajarenc** [comment]
done

**Pull request merged**